appengine/rails-cloudsql-mysql/Gemfile.lock

```diff
@@ -1,297 +1,340 @@
+    rspec-core (3.12.2)
+      rspec-support (~> 3.12.0)
+    rspec-expectations (3.12.3)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.9.0)
-    rspec-mocks (3.9.1)
+      rspec-support (~> 3.12.0)
+    rspec-mocks (3.12.6)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.9.0)
-    rspec-rails (3.9.0)
-      actionpack (>= 3.0)
-      activesupport (>= 3.0)
-      railties (>= 3.0)
-      rspec-core (~> 3.9.0)
-      rspec-expectations (~> 3.9.0)
-      rspec-mocks (~> 3.9.0)
-      rspec-support (~> 3.9.0)
-    rspec-support (3.9.2)
-    rspec_junit_formatter (0.4.1)
+      rspec-support (~> 3.12.0)
+    rspec-rails (6.0.3)
+      actionpack (>= 6.1)
+      activesupport (>= 6.1)
+      railties (>= 6.1)
+      rspec-core (~> 3.12)
+      rspec-expectations (~> 3.12)
+      rspec-mocks (~> 3.12)
+      rspec-support (~> 3.12)
+    rspec-support (3.12.1)
+    rspec_junit_formatter (0.6.0)
       rspec-core (>= 2, < 4, != 2.12.0)
     ruby2_keywords (0.0.5)
-    ruby_dep (1.5.0)
-    sass (3.7.4)
-      sass-listen (~> 4.0.0)
-    sass-listen (4.0.0)
-      rb-fsevent (~> 0.9, >= 0.9.4)
-      rb-inotify (~> 0.9, >= 0.9.7)
-    sass-rails (5.0.7)
-      railties (>= 4.0.0, < 6)
-      sass (~> 3.1)
-      sprockets (>= 2.8, < 4.0)
-      sprockets-rails (>= 2.0, < 4.0)
-      tilt (>= 1.1, < 3)
-    signet (0.17.0)
+    sass-rails (6.0.0)
+      sassc-rails (~> 2.1, >= 2.1.1)
+    sassc (2.4.0)
+      ffi (~> 1.9)
+    sassc-rails (2.1.2)
+      railties (>= 4.0.0)
+      sassc (>= 2.0)
+      sprockets (> 3.0)
+      sprockets-rails
+      tilt
+    signet (0.18.0)
       addressable (~> 2.8)
       faraday (>= 0.17.5, < 3.a)
       jwt (>= 1.5, < 3.0)
       multi_json (~> 1.10)
-    sprockets (3.7.2)
+    sprockets (4.2.1)
       concurrent-ruby (~> 1.0)
-      rack (> 1, < 3)
+      rack (>= 2.2.4, < 4)
     sprockets-rails (3.4.2)
       actionpack (>= 5.2)
       activesupport (>= 5.2)
       sprockets (>= 3.0.0)
-    sqlite3 (1.5.3)
-      mini_portile2 (~> 2.8.0)
+    sqlite3 (1.6.6-x86_64-linux)
     stackdriver (0.21.1)
       google-cloud-error_reporting (~> 0.41)
       google-cloud-logging (~> 2.1)
       google-cloud-trace (~> 0.40)
     stackdriver-core (1.5.0)
       google-cloud-core (~> 1.2)
-    thor (1.2.1)
-    thread_safe (0.3.6)
-    tilt (2.0.10)
-    tzinfo (1.2.10)
-      thread_safe (~> 0.1)
-    web-console (3.7.0)
-      actionview (>= 5.0)
-      activemodel (>= 5.0)
+    thor (1.2.2)
+    tilt (2.3.0)
+    timeout (0.4.0)
+    tzinfo (2.0.6)
+      concurrent-ruby (~> 1.0)
+    web-console (4.2.1)
+      actionview (>= 6.0.0)
+      activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
-      railties (>= 5.0)
-    websocket-driver (0.7.5)
+      railties (>= 6.0.0)
+    webrick (1.8.1)
+    websocket-driver (0.7.6)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
     xpath (3.2.0)
       nokogiri (~> 1.8)
+    zeitwerk (2.6.11)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
Can we add the `ruby` platform to all the Gemfile.lock files, to make sure they'll work on non-linux platforms? (You should be able to `bundle lock --add-platform ruby` to add the platform and reresolve dependencies taking the new platform into account.)

Reply:
Done. Fixed for all files touched for this change.

Reply:
Did you update the lockfile using `bundle` or did you just edit the lockfile with an editor? I'm seeing some of the lockfiles now with platform set to ruby but some of the gems listing their `x86_64-linux` builds.

Reply:
Manually via the editor. Did not use the bundle command.

Reply:
I think we need to use the bundle command, otherwise the overall platform specified in the Gemfile.lock doesn't match the platforms of the gems listed in the same.

Reply:
Done. Added platform via the bundle command. Ensured that I added both x86_64 & ruby and then cleaned out linux based ones.

Reply:
Thanks!